packages/manager/modules/pci/src/projects/project/notebooks/stop-notebook/stop-notebook.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
```

Comment:
Are you sure all these imports are needed ?

Reply:
Looking at the `delete-notebook.module.js` file, it seems that we don't have to import `@ovh-ux/ng-translate-async-loader`,  `angular-translate` and `@ovh-ux/ui-kit`. 
I've removed them those imports here: https://github.com/ovh/manager/pull/6831/commits/86e0de1243f75dd4d3a0a96671f3d2af50b57c3e

Reply:
lgtm then :)